### ISSUE TITLE
Playlist block: User can start from selected audio track, but cannot pause.

### ISSUE THREAD

**Issue by joedolson** [WordPress/gutenberg#75582]
### Description

The playlist block has an overall play/pause button for the playlist, but you can also start play from any track. Once you've started the track, however, you cannot stop the track from the same control; you can only pause by navigating back to the playlist control.

To accommodate this, the screen reader text should be amended to 'Select to play or pause this track'. 

### Step-by-step reproduction instructions

1. Insert a playlist block with multiple tracks.
2. Navigate by keyboard to any track.
3. Press 'space' or 'enter' to start playback.
4. Press 'space' or 'enter' again to pause playback.

This will not work, but is an expected behavior.

### Environment info

Tested in Playground with https://github.com/WordPress/gutenberg/pull/75203

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
